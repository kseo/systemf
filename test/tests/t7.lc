(/\X. \x:X. x) [All X.X->X]
