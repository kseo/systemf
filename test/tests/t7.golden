(\x:(All X.(X->X)).x)